milvus/metadata.csv

```diff
@@ -0,0 +1,387 @@
+metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration,short_name,curated_metric,sample_tags
+milvus.bf_search_cnt.bucket,count,,,,Number of bf search per request,0,milvus,,,
+milvus.bf_search_cnt.count,count,,,,Number of bf search per request,0,milvus,,,
+milvus.bf_search_cnt.sum,count,,,,Number of bf search per request,0,milvus,,,
+milvus.bitset_ratio.bucket,count,,,,Bitset ratio,0,milvus,,,
+milvus.bitset_ratio.count,count,,,,Bitset ratio,0,milvus,,,
+milvus.bitset_ratio.sum,count,,,,Bitset ratio,0,milvus,,,
+milvus.build_info,gauge,,,,Build information of milvus,0,milvus,,,
+milvus.build_latency.bucket,count,,,,Index build latency (s),0,milvus,,,
+milvus.build_latency.count,count,,,,Index build latency (s),0,milvus,,,
+milvus.build_latency.sum,count,,second,,Index build latency (s),0,milvus,,,
+milvus.cache_hit_cnt.bucket,count,,,,Cache hit cnt per request,0,milvus,,,
+milvus.cache_hit_cnt.count,count,,,,Cache hit cnt per request,0,milvus,,,
+milvus.cache_hit_cnt.sum,count,,,,Cache hit cnt per request,0,milvus,,,
+milvus.cgo.active_future_total,gauge,,,,Total number of active futures.,0,milvus,,,
+milvus.cgo.cgo_duration_seconds.bucket,count,,,,Histogram of cgo call duration in seconds.,0,milvus,,,
+milvus.cgo.cgo_duration_seconds.count,count,,,,Histogram of cgo call duration in seconds.,0,milvus,,,
+milvus.cgo.cgo_duration_seconds.sum,count,,second,,Histogram of cgo call duration in seconds.,0,milvus,,,
+milvus.cgo.cgo_queue_duration_seconds.bucket,count,,,,Duration of cgo call in queue.,0,milvus,,,
+milvus.cgo.cgo_queue_duration_seconds.count,count,,,,Duration of cgo call in queue.,0,milvus,,,
+milvus.cgo.cgo_queue_duration_seconds.sum,count,,second,,Duration of cgo call in queue.,0,milvus,,,
```

Comment:
The description field should explain what the difference is between these three metrics.